```
IORawData/CSCCommissioning/src/FileReaderDCC.cc
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_0_X_2021-04-27-1100/src/IORawData/CSCCommissioning/src/FileReaderDCC.cc:20:27: warning: 'bzero' call operates on objects of type 'unsigned short' while the size is based on a different type 'unsigned short *' [-Wsizeof-pointer-memaccess]
```

--- a/IORawData/CSCCommissioning/src/FileReaderDCC.cc
+++ b/IORawData/CSCCommissioning/src/FileReaderDCC.cc
@@ -17,7 +17,7 @@ FileReaderDCC::FileReaderDCC(void) {
     throw std::runtime_error(std::string("Wrong platform: sizeof(unsigned long long)!=8 || sizeof(unsigned short)!=2"));
   raw_event = new unsigned short[200000 * 40];
   end = (file_buffer_end = file_buffer + sizeof(file_buffer) / sizeof(unsigned long long));
-  bzero(raw_event, sizeof(raw_event));
+  bzero(raw_event, sizeof(raw_event) * 200000 * 40);
   bzero(file_buffer, sizeof(file_buffer));
   word_0 = 0;
   word_1 = 0;

